pkgs/top-level/python-packages.nix

```diff
@@ -29464,6 +29464,25 @@ EOF
     };
   };
 
+  pywbem = buildPythonPackage rec {
+    name  = "pywbem-${version}";
+    version = "0.10.0";
+    src = pkgs.fetchFromGitHub {
+      owner = "pywbem";
+      repo = "pywbem";
+      rev = "v${version}";
+      sha256 = "0jcwklip03xcni0dvsk9va8ilqz21g4fxwqd5kzvv91slaadfcym";
+    };
+    propagatedBuildInputs = with self; [ m2crypto ply pyyaml six ];
+    doCheck = false;
```

Comment:
Actually, now that I have your attention, I was hoping you could clarify something... ;-) If we run the tests, they don't actually run and I am told that 0 tests ran. I have tried adding tox and pytest to ```checkInputs``` but that doesn't change anything. Is there anything obvious I am missing to actually run the tests?